### PR TITLE
Web ui test admin general settings updater settings

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
This PR adds tests for:
- updater settings
- cron selection
- log level selection

Related issue: #32851

## How Has This Been Tested?
Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/lib/AdminGeneralSettingsPage.php [hunk 1]
```diff
@@ -170,4 +178,63 @@ public function setLegalUrl($legalUrlType, $legalUrlValue) {
...
+
+	/**
+	 * set cron job value
+	 *
+	 * @param string $cronJob
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	public function setCornJobValue($cronJob) {
```
Comment (phil-davis) [inline tests/acceptance/features/lib/AdminGeneralSettingsPage.php]: s/Corn/Cron/g

File: tests/acceptance/features/lib/AdminGeneralSettingsPage.php [hunk 2]
```diff
@@ -170,4 +178,63 @@ public function setLegalUrl($legalUrlType, $legalUrlValue) {
...
+		} else {
+			throw new \Exception(
+				__METHOD__ . " invalid cron job type: $cronJob"
+			);
+		}
+		if ($selectCorn === null) {
+			throw new ElementNotFoundException(
+				__METHOD__ .
+				" xpath $selectCorn->getXpath() " .
+				"could xpath for radio button"
```
Comment (phil-davis) [inline tests/acceptance/features/lib/AdminGeneralSettingsPage.php]: could not find xpath...

**Comment by paurakhsharma** [comment]
@phil-davis, I have made the necessary changes. Can you please review it again.

**Review by phil-davis — APPROVED** [review]

**Comment by paurakhsharma** [comment]
Backport on: #32992

**Pull request merged**